package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pillarx",
-  "version": "0.1.10",
+  "version": "0.1.11",
```

Comment:
Definitely a minor bump - we've introduced features 🙏

Reply:
makes sense, changed to `0.2.0`